wilds/common/metrics/metric.py

```diff
@@ -203,7 +203,7 @@ def compute_element_wise(self, y_pred, y_true, return_dict=True):
         """
         element_wise_metrics = self._compute_element_wise(y_pred, y_true)
         batch_size = y_pred.size()[0]
-        assert element_wise_metrics.dim()==1 and element_wise_metrics.numel()==batch_size
+        assert (element_wise_metrics.dim()==1 and element_wise_metrics.numel()==batch_size) or (element_wise_metrics.dim()==2 and element_wise_metrics.size(0)==batch_size)
```

Comment:
Since ElementwiseMetric is supposed to return Tensors of size (batch size, ), I think it's better to use a different Metric subclass (probably MultiTaskMetric). Can you revert to the original assert?